src/components/calculator.js

```diff
@@ -1,45 +1,64 @@
 import React, { Component } from 'react';
+import calculate from './logic/calculator';
```

Comment:
- [ ] [ OPTIONAL ] Instead of using both file names as calculator.js, It will be confusing. Kindly change it to /logic/calculate. & change the file name in the `logic` folder to `calculate.js`